src/app/layout.tsx

```diff
@@ -1,9 +1,24 @@
 import type { Metadata } from 'next';
-import { Inter } from 'next/font/google';
-import '@/styles/globals.css';
 import Providers from './providers';
+import LocalFont from "next/font/local";
+import '@/styles/globals.css';
 
-const inter = Inter({ subsets: ['latin'] });
+const fontDana = LocalFont({
+  src: [
+    {
+      path: "../assets/fonts/dana/Dana-Regular.woff",
+      weight: "400",
+      style: "normal",
+    },
+    {
+      path: "../assets/fonts/dana/Dana-Bold.woff",
+      weight: "600",
+      style: "normal",
+    },
+  ],
+  style: "normal",
+  display: "block",
```

Comment:
The default value of `style` & `display` props is optimal. I think we can omit them.